ocaml/idl/datamodel_host.ml

```diff
@@ -1374,6 +1374,13 @@ let create_params =
     ; param_release= numbered_release "25.32.0"
     ; param_default= Some (VBool false)
     }
+  ; {
+      param_type= Map (String, String)
+    ; param_name= "software_version"
+    ; param_doc= "Information about the software versions on the host"
```

Comment:
Do you need to implement something in xapi_upgrade, so that this is not immediately empty on upgrade? At least for the coordinator.

Reply:
@edwintorok This defines a function parameter, not a field. And this is only used on pool join.